### PR TITLE
Adds in Welding Gas Mask

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Ports over the welding gas mask from TG....meant to do this literally 10 months ago...but forgot/put it off.

- Combines the funcitonality of a gas mask and a welding helmet into one.
- Low level R&D item.

### CODE REVIEW

**Comment by Spacemanspark** [comment]
Oh God I'd forgotten about that item.
:+1:

File: code/modules/clothing/masks/gasmask.dm [hunk 1]
```diff
@@ -14,6 +14,50 @@
...
+	origin_tech = "materials=2;engineering=2"
+	action_button_name = "Toggle Welding Helmet"
+
+/obj/item/clothing/mask/gas/welding/attack_self()
+	toggle()
+
+
+/obj/item/clothing/mask/gas/welding/verb/toggle()
+	set category = "Object"
+	set name = "Adjust welding mask"
```
Comment (tigercat2000) [inline code/modules/clothing/masks/gasmask.dm]: I'm 90% sure that nothing else has it's adjust verb anymore, and it's supposed to only have an action button.

Comment (Fox-McCloud) [inline code/modules/clothing/masks/gasmask.dm]: https://github.com/ParadiseSS13/Paradise/blob/0ec14cf267c4695d86dd8cb7cada5c779573d1b5/code/modules/clothing/glasses/glasses.dm#L213

they all have their verbs

Comment (tigercat2000) [inline code/modules/clothing/masks/gasmask.dm]: @Fox-McCloud Huh. Must be thinking of something else.

**Pull request merged**